src/toil/provisioners/aws/awsProvisioner.py

```diff
@@ -269,11 +269,11 @@ def addNodes(self, nodeType, numNodes, preemptable, spotBid=None):
                 # the biggest obstacle is AWS request throttling, so we retry on these errors at
                 # every request in this method
                 if not preemptable:
-                    logger.info('Launching %s non-preemptable nodes', numNodes)
```

Comment:
This can be up for debate, but I think information about which and how many nodes are running is useful information to be log info. That would apply at least for these next few changes you've made.